### PR TITLE
README/Dependencies - Updated "Quickstart Guide"

### PR DESCRIPTION
**Pull request by JakeMercer**
Updated the Quickstart Guide section of the README as some deviations were
required when following this with a fresh Ubuntu 16.04 install.  Fixes to
''requirements.txt'' were required to remove the "futures" dependency which is
in the python3 standard library and is only intended for use with python2;
"fasm" now goes by "pyfasm".

Signed-off-by: Jake Mercer <jmercer04@qub.ac.uk>

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -1,5 +1,4 @@
-fasm
-futures
+pyfasm
```
Comment (litghost) [inline requirements.txt]: This is incorrect, as fasm is referring to https://github.com/SymbiFlow/fasm not to https://github.com/srounet/pyfasm.  Maybe updated it something like https://stackoverflow.com/questions/16584552/how-to-state-in-requirements-txt-a-direct-github-source ?

File: README.md [hunk 2]
```diff
@@ -46,26 +46,32 @@ Build the C++ tools:
     make build
 
 ### Step 5: ###
+Alias python to python3 to avoid using python2 accidentally:
+
+    alias python=python3
```
Comment (litghost) [inline README.md]: Why was this required?  Are there commands in prjxray that still reference python (rather than python3)?

Comment (JakeMercer) [reply README.md]: You're right, I don't think this is necessary; I will remove it.
